setup.py

```diff
@@ -70,6 +70,7 @@
     python_requires=">=3.6",
     extras_require={
         'cplex': ["cplex; python_version < '3.9'"],
+        'cvx': ['cvxpy>1.0.0,!=1.1.0,!=1.1.1,!=1.1.2'],
```

Comment:
Is this cvx version list what is needed for the logic here or was it just inherited from what we had before that I think was done to be the same as elsewhere in qiskit. I think I would set this to what will work for the code here and let the install process figure an apprropriate version given other cvx dependencies. I just looked at Ignis and its has a version dependency but it simply needs one greater than some specific version.

Reply:
This was copied from Aqua as is. Honestly, no idea why it is so.

Reply:
@manoelmarques I believe this set of versions came from elsewhere in qiskit which was using it in the past. The exclusions, as far as I recall, are because of problems with the set of function that was used by that code in those releases. The list was added to aqua to be consistent with versions supported. Ignis simply states >= 1.0.15 for a version here since presumably some function was added/changed then and that as a min is needed. If there is no specific version requirement for the functionality of cvxpy used here we could drop any version list or just use the same as Ignis since any install that included Ignis would need at least that level anyway.

Reply:
I removed the version list on cvxpy and added cvxpy install on CI.